Translation.Rmd

```diff
@@ -171,8 +171,12 @@ We could list all the possible attributes of the `<p>` tag in the function defin
 ```{r named}
 dots_partition <- function(...) {
   dots <- list2(...)
-
+  
   is_named <- names(dots) != ""
+  if (length(is_named) == 0) {
```

Comment:
I think it would be slightly clearer to write:

```R
if (is.null(names(dots))) {
  is_named <- rep(FALSE, length(dots))
} else {
  is_named <- names(dots) != ""
}
```

Reply:
Yes, it looks better.